at_server_status/lib/at_status_impl.dart

```diff
@@ -23,11 +24,12 @@ class AtStatusImpl implements AtServerStatus {
     _root_port = value;
   }
 
-  AtStatusImpl({String? rootUrl, int? rootPort}) {
+  AtStatusImpl(String atSign, {String? rootUrl, int? rootPort}) {
     rootUrl ??= 'root.atsign.org';
     _root_url = rootUrl;
     rootPort ??= 64;
     _root_port = rootPort;
+    atLookupImpl = AtLookupImpl(atSign, _root_url!, _root_port!);
```

Comment:
if user calls constructor for an atsign multiple times, same instance should be returned